dashboard/config/scripts/levels/ErinPoembot.level

```diff
@@ -0,0 +1,116 @@
+<GamelabJr>
```

Comment:
i don't think we want to commit this level, right?

Reply:
Yup! Thanks for catching.